src/realTimeReporter/utils.ts

```diff
@@ -11,6 +12,15 @@ export const subscribeToEvent = (event: EVENTS, callback: (params: any) => void)
   process.on(event, callback);
 };
 
+export const ejectFileFromLog = (logData: LogRQ): { log: LogRQ, file?: Attachment } => {
+  const { file, ...log } = logData;
+
+  return {
+    log,
+    file: file ? { type: FILE_TYPES.JPG, ...file } : undefined,
```

Comment:
NodeJS 8.2.1 doens't support object spread/rest operator. Which NodeJS version this agent should support?

Reply:
It will use `Object.assign` after ts compilation to `.js`

Reply:
It will use `Object.assign` after ts compilation to `.js`